db-client-java/src/main/java/com/eventstore/dbclient/OptionsBase.java

```diff
@@ -5,10 +5,15 @@
 class OptionsBase<T> {
     protected Timeouts timeouts;

```

Comment:
**CONCERN:** Shouldn't we remove timeouts and use deadlines now? Also, I think that we should update default values.